test/functional/Valhalla/src/org/openj9/test/lworld/ValueTypeGenerator.java

```diff
@@ -46,8 +46,91 @@
 	static {
 		generator = new ValueTypeGenerator();
 	}
-	
-	private static byte[] generateClass(String className, String[] fields, boolean isVerifiable, boolean isRef) {
+
+	private static class ClassConfiguration {
```

Comment:
can you add a comment describing what `accessedContainer` and `valueClassUsedInCode` are

Reply:
Addressed in commit fbba19114